requirements/dev.txt

```diff
@@ -22,3 +22,4 @@ tox>=3.23.0,<4.0.0
 troposphere>=2.0.0,<2.1.0
 twine>=1.12.1,<2.0.0
 wheel==0.32.3
+ipdb==0.13.7
```

Comment:
Added by mistake?

Reply:
@mrowlingfox No, I just thought it's convenient to have the debugger available in dev.txt. It's not a must-have obviously.